project_profitability_fields/views/project_project.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="project_custom_fields_form_view" model="ir.ui.view">
+        <field name="name">project.project.form</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='tag_ids']" position="after">
+                <field name="project_date"/>
+                <field name="estimated_cost"/>
+                <field name="margin" string="Margin(%)"/>

```

Comment:
We should consider using `widget="percentage"` instead. That widget should format this data as we desire, but (I'm not sure) that data is stored different, so you should change python calculations (like a `25 %` value is actually stored as `0.25`, not `25.0`)